requirements.txt

```diff
@@ -0,0 +1 @@
+codespell==2.3.0
```

Comment:
Should we add this to the renovabot upgrades to check for new versions?

Reply:
added to the renovate.json